workflows/qc/qc/config.vsh.yaml

```diff
@@ -0,0 +1,101 @@
+functionality:
+  name: "qc"
+  namespace: "qc"
```

Comment:
When switching to viash 0.8.0,  it's best to move workflows managed by Viash to `src/workflows`

```suggestion
  name: "qc"
  namespace: "workflows"
```

Reply:
Would you be okay with tackling this in a separate PR (updating to viash 0.8.0)?

Reply:
sure